docs/sequence_diagrams/notification-actionFlow.plantuml

```diff
@@ -0,0 +1,108 @@
+@startuml
```

Comment:
Please, apply all that is applicable from the review of the previous diagrams.

Reply:
Will action once the renaming is agreed on.